extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -72,7 +77,7 @@ public function index() {
 		// Inject the JS
 		$srcs = AssetsManager::getInstance()->getGroupCommonURL( 'app_promo_landing_js' );
 		foreach( $srcs as $src ) {
-			$this->wg->Out->addScript( "<script type=\"{$wgJsMimeType}\" src=\"{$src}\"></script>" );
+			$this->wg->Out->addScript( "<script type=\"{$this->wg->JsMimeType}\" src=\"{$src}\"></script>" );
```

Comment:
What do you think about simply using `addAsset` https://github.com/Wikia/app/blob/2eb1f6e83201550ff4e0be07f02bfdc69c44ee6e/includes/wikia/nirvana/WikiaResponse.class.php#L512-L512 ?